tests/test_external_account.py

```diff
@@ -508,6 +512,7 @@ def test_with_scopes_full_options_propagated(self):
             token_url=self.TOKEN_URL,
             credential_source=self.CREDENTIAL_SOURCE,
             service_account_impersonation_url=self.SERVICE_ACCOUNT_IMPERSONATION_URL,
+            service_account_impersonation_options={},
```

Comment:
why is not having the lifetime value? This test is for all options?

Reply:
same for other full options tests

Reply:
Done